src/Tezos/Forging.cpp

```diff
@@ -84,14 +84,14 @@ Data forgeOperation(const Operation& operation) {
         auto publicKey = PublicKey(operation.reveal_operation_data().public_key(), TWPublicKeyTypeED25519);
         auto forgedPublicKey = forgePublicKey(publicKey);
 
-        /* Uncomment for debugging */
+        /* Uncomment for debugging
```

Comment:
Can we remove comments for debugging? It’s not document either

Reply:
It helped me a lot when I was having issues to forge those binaries.

Reply:
yes, I'm used to have them in my git stash, git stash pop if you want to debug them